### ISSUE TITLE
Failed uploads with excessive output

### ISSUE THREAD

**Issue by stuarta0** [laurent-clouet/sheepit-client#114]
Looks like people aren't sticking to the "you should keep your renders < 20min and < 40MB per frame". Just rendered a frame for 2hrs and the output was a 7500x3750 16 bit PNG that weighed in at 165MB and crashed on upload. This wipes out the client with an OutOfMemoryException. 

Possibly need some blacklisting/messaging code as suggested in issue #87 to inform the server or project owner that they're exceeding the recommended limits. Plus I get no points for spending all that energy rendering a frame that can't be uploaded. And I can't render anyone else's work (until I restart it) because this project crashed the client.

```
Exception in thread "main" java.lang.OutOfMemoryError: Java heap space
        at java.util.Arrays.copyOf(Unknown Source)
        at java.io.ByteArrayOutputStream.grow(Unknown Source)
        at java.io.ByteArrayOutputStream.ensureCapacity(Unknown Source)
        at java.io.ByteArrayOutputStream.write(Unknown Source)
        at sun.net.www.http.PosterOutputStream.write(Unknown Source)
        at java.io.DataOutputStream.write(Unknown Source)
        at com.sheepit.client.Server.HTTPSendFile(Server.java:681)
        at com.sheepit.client.Client.confirmJob(Client.java:747)
        at com.sheepit.client.Client.run(Client.java:328)
        at com.sheepit.client.standalone.GuiText.start(GuiText.java:74)
        at com.sheepit.client.standalone.Worker.doMain(Worker.java:295)
        at com.sheepit.client.standalone.Worker.main(Worker.java:95)
```

**Comment by laurent-clouet** [comment]
Hello,
There is no rule for frame size but i agree 165MB per frame is way too big.
Do you have more log, i'm looking for the job id (or name) so i can check it out and most likely block it.

**Comment by stuarta0** [comment]
I did get the job ID but it's on my Windows machine at home. I'll be back there in 8hrs. It happens daily at the moment - whether it's the same project or others I haven't logged their id's.

FYI I'm referring to this statement regarding output (maybe I misread the 20min suggestion somewhere else):
https://www.sheepit-renderfarm.com/faq.php

Does Sheep It! support OpenEXR as output file?
In theory yes BUT we disabled it due to the size of the output file, a 1920x1080 frame can take more than 40MB, it will be very difficult for the renderer to upload it. For the owner, it will be even worse; a 400 frames animation could be 16GB to download.
